### PR TITLE
Run in cluster

### PR DESCRIPTION
**Pull request by jerith**

### CODE REVIEW

**Comment by jerith** [comment]
I've tested the Dockerfile and the deploy YAML manually in minikube, so those should be okay.

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+      - 'v*'
+
+jobs:
+  release:
+    name: release
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Publish Docker Image to Github packages
+      uses: machine-learning-apps/gpr-docker-publish@master
```
Comment (bearnard) [inline .github/workflows/release.yaml]: do we want to pin this?

**Comment by bearnard** [comment]
👍

File: .github/workflows/release.yaml (hunk 1 above)
Comment (jerith) [reply .github/workflows/release.yaml]: Maybe? I copied it from your release flow and changed the image name. :-P

Comment (bearnard) [reply .github/workflows/release.yaml]: ah, we probably should for a number of obvious reasons, I'll update mine too.

**Comment by bearnard** [comment]
👍

**Comment by bearnard** [comment]
👍

**Pull request merged**